src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandler.java

```diff
@@ -199,6 +200,10 @@ private boolean hasPhmeRequestOrConfidentialityUnderReview(SscsCaseData sscsCase
         return !response.getErrors().isEmpty();
     }
 
+    private boolean isPhmeAbleBenefitTypeCase(SscsCaseData sscsCaseData) {
+        return !sscsCaseData.isBenefitType(Benefit.CHILD_SUPPORT);
```

Comment:
The method name of this one doesn't really make sense to me considering it's just checking if it is not a child support case. I've tried to get my head round this whole logic a few times and im pretty confused as to what is going on. Is there anyway to make clearer? Or if not, maybe put a comment as to what this is actually doing?